packaging/installer.iss

```diff
@@ -16,18 +26,23 @@ DefaultDirName="{commoncf64}\VST3\{#ProductName}.vst3"
 DisableDirPage=yes
 
 ; MAKE SURE YOU READ THE FOLLOWING!
-LicenseFile="EULA"
+LicenseFile="..\LICENSE"
```

Comment:
I recently added the EULA as a discrete entity since for most people’s projects, the pamplejuce MIT license is probably not desirable...

Reply:
Gotcha, reverted that line!